### ISSUE TITLE
[Switch] Component does not render name prop

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-react#1330]
Reported by client (Ticket ID: 1567622)

Setting the name prop of the Switch component does not render a name for the input.

Steps to reproduce:

1) Inspect the input element of the Switch component
2) Notice that even though the name prop has been specified, the input does not have a name rendered

https://stackblitz.com/edit/react-gjjnuw?file=app/main.tsx

**Comment by iadnonov** [comment]
The issue has been resolved in version [6.1.0](https://www.telerik.com/kendo-react-ui/components/changelogs/ui-for-react/#v6.1.0).
